test/unit/uk/gov/hmrc/tariffclassificationfrontend/connector/EmailConnectorSpec.scala

```diff
@@ -75,11 +56,16 @@ class EmailConnectorSpec extends UnitSpec
       stubFor(post(urlEqualTo(s"/templates/${EmailType.COMPLETE}"))
         .withRequestBody(new EqualToJsonPattern(fromResource("parameters_email-request.json"), true, false))
         .willReturn(aResponse()
-            .withBody(fromResource("email_template-response.json"))
+          .withBody(fromResource("email_template-response.json"))
           .withStatus(HttpStatus.SC_OK))
       )
 
       await(connector.generate(email)) shouldBe EmailTemplate("text", "html", "from", "subject", "service")
+
+      verify(
+        postRequestedFor(urlEqualTo(s"/templates/${EmailType.COMPLETE}"))
+          .withoutHeader("X-Api-Token")
```

Comment:
calls to microservices not belonging to us should NOT contain the extra header we inject